app/helpers/applications_helper.rb

```diff
@@ -24,6 +24,12 @@ def scraped_and_received_text(application)
     text
   end
 
+  def decode(description)
+    coder = HTMLEntities.new
+    first_decoding = coder.decode(description)
```

Comment:
That's really helpful @hisayohorie :+1: I don't know of any better ways off the top of my head but this would be a great place to put a comment to explain why we're calling this twice.

Whenever code is difficult to understand first try to simplify it so it is easy to understand :smile: but if you can't then that's when a comment is the way to go :ok_hand: